components/ReviewCard.test.js

```diff
@@ -334,4 +334,21 @@ describe('ReviewCard Component', () => {
       )
     ).toBeVisible()
   })
+  test('Should render Accordion with shadow when Accordion header is clicked', () => {
```

Comment:
description should match the tests: `Accordion should be visible when accordion header is clicked`